packages/mui-material/src/Autocomplete/Autocomplete.js

```diff
@@ -572,6 +575,9 @@ const Autocomplete = React.forwardRef(function Autocomplete(inProps, ref) {
     className: classes.paper,
   });
 
+  const groupedOptionsDeferred = useDeferredValue(popupOpen ? groupedOptions : EMPTY_ARRAY);
```

Comment:
This should be in the hook version, no? So "Base UI" users can have the same benefit
```suggestion
```